graftm/hmmer.py

```diff
@@ -511,10 +502,10 @@ def _get_read_names(self, search_result, max_range):
             alignment span.
         max_range : int
             Maximum range that a gene can extend within a contig. Any hits 
-            that extend beyond this length cannot be linked. max_range is defined 
-            as 1.5 X the average length of all full length genes used in the 
-            search database. This is defined in the CONTENTS.json file within a 
-            gpkg.
+            that extend beyond this length cannot be linked. max_range is 
+            defined as 1.5 X the average length of all full length genes used 
```

Comment:
Okay, I changed it to "set" to avoid ambiguity.

It's used below on line 570 and 574